code-challenges.test.js

```diff
@@ -31,10 +42,28 @@ const colors2 = [
 
 // Pseudo code:
 
+// 1.) I am creating a function that is named colorArray and setting it to take in an array. Have the placholder array as a parameter.
+
+const colorArray = (array) => {
+
+
+// 2.) Return have it return the array and the test will pass.
```

Comment:
The test passes because you are only testing that the output array is an array containing everything that the original array had minus the first element. This means that returning the original array will meet those conditions, and the test will pass, even though you didn't do anything in the function. To test that the function is doing what it is supposed to do, you will need to add additional test assertions. For example:
`expect(colorArray(colors1)).not.toContain(colors1[0])`.
This assertion ensures that the output array does not contain the first element in the argument.